website/content/docs/known-issues-and-solutions.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Known Issues and Solutions
+weight: 20
+group: support
+---
+
+## Background
+
+There are a couple of issues that users face while integrating or working with Netlify CMS and while the community is doing their best fixing them, here is a list of known issues and their solutions.
+
+## API Errors
+
+A list of error messages and their possible cause(s) to provide more context while debugging.
+
+#### 1. Error Message.
+
+```
+API_ERROR: {"message":"404 File Not Found"}
+```
+
+#### Possible Causes.
+
+- The collection folder or file property is missing a value (a dedicated file or folder, can't be blank or root). The file or folder should be relative to root (so `src/posts` for example).
+- There are subfolders within the collection folder.
+
+#### 2. Error Message
+
+```
+API_ERROR: Not Found
+```
+
+#### Possible Cause
+
+- The media library folder is missing, or the value of the media library property within config.yml isn't correct.
+
+#### 3. Error Message
+
+```
+Failed to load entries: API_ERROR: Not Found
+```
+
+#### Possible Causes
+
+- Git Gateway is not enabled: Enable Git Gateway from the Netlify Identity settings or refresh the token.
+- There aren't any entries for a given collection. Not sure if we should throw this error at all actually.
+
+#### 4. Error Message
+
+```
+Failed to load entry: API_ERROR: Not Found
+```
+
+#### Possible Causes
+
+- The collection folder or file property is missing a value (a dedicated file or folder, can't be blank or root). The file or folder should be relative to root (so `src/posts` for example).
```

Comment:
The first cause here is also under `1. Error Message.` and the second one shouldn't happen anymore.